### PR TITLE
feature/session wizard this closes #82  also adds feature/loader this closes #115

### LINKED ISSUE #82: Component: Session Wizard

### LINKED ISSUE #115: Component: Loader

### PR DESCRIPTION
**Pull request by Andreas-sjolund-chas**
- Add session wizard
- Add a loader component

The wizardpage can be found on route "/createsession" for now. 
This will be changed when we implement redux and can call something like "open modal" from different places.

### CODE REVIEW

File: src/Components/CreateSession/End.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import Button from "../../Elements/Button";
+import Input from "../../Elements/Input";
+import InputWithIcon from "../../Elements/InputWithIcon";
+import Icon from "../../Elements/Icon";
+import CopyTextfield from "../../Elements/CopyTextfield";
+
+const End = ({ styles }) => (
+  <FlexContainer align="start">
+    <Heading appearance="primary" size="2">
+      Have a greate session!
```
Comment (robertjarske) [inline src/Components/CreateSession/End.js]: What is greate? What a GREAT word ;)

**Review by robertjarske — CHANGES_REQUESTED** [review]

**Review by robertjarske — APPROVED** [review]

File: src/Components/Wizard.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+              onClick={
+                this.state.currentPage === 2
+                  ? this.dummyFunction
+                  : this.nextPage
+              }
+              appearance="secondary"
+            >
+              {this.state.currentPage === 2
+                ? "FINISH - " + (this.state.currentPage + 1) + " / 3"
+                : "NEXT - " + (this.state.currentPage + 1) + " / 3"}
```
Comment (lessp) [inline src/Components/Wizard.js]: Could use children.length or something in order to calculate the number of pages.

File: src/Components/Wizard.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    /** Handle the finished settings */
+    console.log(e.target);
+  }
+
+  dummyFunction(e) {
+    /** This is only used to prevent the button for breaking during storybook */
+    e.preventDefault();
+  }
+
+  /** TODO: Set up redux with each input from each step so we can handle all the settings at once */
```
Comment (lessp) [inline src/Components/Wizard.js]: Focus on just being able to pass the data from the Wizard up to "something" else.

**Review by lessp — CHANGES_REQUESTED** [review]

**Review by lessp — APPROVED** [review]

**Review by Ciavarella — APPROVED** [review]
Looks nice!

**Pull request merged**